starboard/android/shared/audio_sink_min_required_frames_tester.cc

```diff
@@ -79,17 +80,19 @@ void MinRequiredFramesTester::AddTest(
 void MinRequiredFramesTester::Start() {
   SB_DCHECK(thread_checker_.CalledOnValidThread());
   // MinRequiredFramesTester only supports to start once.
-  SB_DCHECK(!SbThreadIsValid(tester_thread_));
+  SB_DCHECK(tester_thread_ == 0);
 
-  tester_thread_ =
-      SbThreadCreate(0, kSbThreadPriorityLowest, kSbThreadNoAffinity, true,
-                     "audio_track_tester",
-                     &MinRequiredFramesTester::TesterThreadEntryPoint, this);
-  SB_DCHECK(SbThreadIsValid(tester_thread_));
+  pthread_create(&tester_thread_, nullptr,
+                 &MinRequiredFramesTester::TesterThreadEntryPoint, this);
+  SB_DCHECK(tester_thread_ != 0);
 }
 
 // static
 void* MinRequiredFramesTester::TesterThreadEntryPoint(void* context) {
+  ::starboard::shared::pthread::ThreadSetPriority(kSbThreadPriorityLowest);
```

Comment:
Just curious on whether we're planning to migrate ThreadSetPriority to pthread too?

Reply:
Yes, however it would take a few CLs to get there.